src/main/java/com/snowflake/kafka/connector/Utils.java

```diff
@@ -924,6 +924,19 @@ public static String getExceptionMessage(String customMessage, Exception ex) {
     return formatString(GET_EXCEPTION_FORMAT, customMessage, message, cause);
   }
 
+  public static SnowflakeSinkConnectorConfig.BehaviorOnNullValues getBehaviorOnNullValuesEnum(
+      Map<String, String> config) {
+    return Arrays.stream(SnowflakeSinkConnectorConfig.BehaviorOnNullValues.values())
+        .filter(
+            behavior ->
+                behavior
+                    .toString()
+                    .equalsIgnoreCase(
```

Comment:
needed because we need to convert the config param to the enum with an `equalsIgnoreCase`.

Alternatively I can do something like the following since we only have two options in the enum, however it's not optimal if we decide to add any more behaviors

```
return config.get(BEHAVIOR_ON_NULL_VALUES_CONFIG).equalsIgnoreCase(SnowflakeSinkConnectorConfig.BehaviorOnNullValues.IGNORE.toString()) ?
        SnowflakeSinkConnectorConfig.BehaviorOnNullValues.IGNORE :
        SnowflakeSinkConnectorConfig.BehaviorOnNullValues.DEFAULT;
```
return